client/styles/_lists.scss

```diff
@@ -3,47 +3,4 @@ ol {
   margin: 0;
```

Comment:
Delete this as well (i mean the entire file). This was the initial problem, we don't want to override the styles anymore.

Basically there should be no default styles for `ul` or `ol`.

PS: have another look at the List storybook after you delete this file and make sure it looks ok.